### PR TITLE
Fixes #9649 - Add 7.4 release notes

### LINKED ISSUE #9649: Create release notes for 7.4-preview.1
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Document new `Get-Foo` cmdlet" instead of "New cmdlet."

### Summary

- Create release notes article to summarize what's new
- Update Get-WhatsNew cmdlet

### Details

_No response_

### Articles

- reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md

### Related Source Pull Requests

- https://github.com/PowerShell/PowerShell/releases/tag/v7.4.0-preview.1

### Related Source Issues

n/a

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#58209](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/58209) - Fixes #9649 - Add 7.4 release notes

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ms.date: 01/05/2023
+---
+
+# What's New in PowerShell 7.4 (preview)
+
+PowerShell 7.4-preview.1 includes the following features, updates, and breaking changes.
+
+For a complete list of changes, see the [Change Log][07] in the GitHub repository.
+
+## Breaking Changes and Improvements
```
Comment (michaeltlombardi) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md:13]: nit: Until we have some improvements, maybe best to say only changes here?

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+- Add completion for values in comparisons when comparing Enums ([#17654][17654]) (Thanks
+  @MartinGC94!)
+
+## Cmdlet and engine improvements
+
+Update to WebCmdlets
+
+- WebCmdlets get **Retry-After** interval from response headers if the status code is 429
+  ([#18717][18717]) (Thanks @CarloToso!)
+- Web cmdlets set default charset encoding to UTF8 ([#18219][18219]) (Thanks @CarloToso!)
```
Comment (michaeltlombardi) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md:37]: Standardizing to the spacing/capitalization used above this line.
```suggestion
- WebCmdlets set default charset encoding to UTF8 ([#18219][18219]) (Thanks @CarloToso!)
```

**Review by michaeltlombardi — COMMENTED** [review]
One fix and two nits, otherwise LGTM!

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md:13]: Agreed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md:37]: Will fix

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+Other cmdlets
+
+- `Update-Help` now reports an error when using implicit culture on non-US systems.
+  ([#17780][17780]) (Thanks @dkaszews!)
+- `Stop-Transcript` now adds `SupportsShouldProcess` ([#18731][18731]) (Thanks @JohnLBevan!)
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md:43]: Will reword

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**